tests/test_problem_bank_helpers_other_functionality.py

```diff
@@ -0,0 +1,68 @@
+from src.problem_bank_helpers import __version__
+from src.problem_bank_helpers import problem_bank_helpers as pbh
+from collections import defaultdict
+import numpy as np
+import pandas as pd
+import os
+import pytest
+import pathlib
+import csv
+
+def test_version():
+    assert __version__ == '0.1.14'
+
+files = sorted(
+    [
+        file.name
+        for file in pathlib.Path(
+            "data/"
+        ).iterdir()
+    ]
+)
+
+files = [("data/" + f) for f in files if f != 'empty.csv']
```

Comment:
```suggestion
files = [("src/problem_bank_helpers/data/" + f) for f in files if f != 'empty.csv']
```
Same idea as my comment for line 18